src/bug-filing/azure-boards/azure-boards-settings-form.tsx

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Dropdown, IDropdownOption, TextField } from 'office-ui-fabric-react';
+import * as React from 'react';
+
+import { NamedSFC } from '../../common/react/named-sfc';
+import { SettingsFormProps } from '../types/settings-form-props';
+import {
+    AzureBoardsBugFilingService,
+    AzureBoardsBugFilingSettings,
+    AzureBoardsIssueDetailField,
+    AzureBoardsIssueDetailLocationDropdownOption,
+} from './azure-boards-bug-filing-service';
+
+export const AzureBoardsSettingsForm = NamedSFC<SettingsFormProps<AzureBoardsBugFilingSettings>>('AzureDevOpsSettingsForm', props => {
+    const options: AzureBoardsIssueDetailLocationDropdownOption[] = [
+        {
+            key: 'reproSteps',
+            text: 'Repro steps (default)',
+        },
+        {
+            key: 'description',
+            text: 'Description',
+        },
+    ];
+
+    const onProjectURLChange = (event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, newValue?: string) => {
```

Comment:
Per bug 1512520 we'll likely want this behavior to be different between the settings panel and the "Specify issue filing location" dialog. It would be great if we could hide that difference from bug filing services and forms.

Reply:
Not taking care of that bug on this PR... sounds like something we want to take care of as part of wiring up.